package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "fxa-geodb",
+  "version": "0.0.2",
+  "private": true,
+  "description": "Firefox AccountsGeoDB Repo for Geolocation based services",
+  "main": "src/fxa-geodb.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "setup": "npm install && bower install",
+    "build": "grunt build && node scripts/max-loader.js &",
```

Comment:
`build` seems like the wrong command because we aren't actually building anything.